install/source/index.markdown

```diff
@@ -74,7 +74,15 @@ title: MoveIt 1 Source Build - Linux
             pip3 install git+https://github.com/catkin/catkin_tools.git
         </code>
     </div>
-    Then re-run <span class="ros-command">catkin config</span>.
+    Then re-run <span class="ros-command">catkin config</span>.</p>
+    <p><b>Note October 2020</b> For Noetic there is currently an issue with the dependency <a href="https://github.com/ros-planning/moveit/pull/2359">fcl</a> and the solution is to use the package in ros-testing:
```

Comment:
I wouldn't talk about an issue:
```suggestion
    <p><b>Note October 2020</b> Until the next ROS release sync, you will need to install ros-noetic-fcl manually from the ros-testing repositories:
```
If possible, we should link to the discourse page, announcing the sync...
Not sure, the _temporary_ status of that issue justifies the effort for this workaround.

Reply:
Ok, good point.  I'm drafting the discourse post about this release I plan on posting after we finish getting moveit in ros-testing.  I'll accept this change and then fix up this to point to the release anouncement where I'll make meantion of the need to do this.